coral/src/app/layout/main-navigation/MainNavigation.test.tsx

```diff
@@ -292,4 +222,37 @@ describe("MainNavigation.tsx", () => {
       });
     });
   });
+
+  describe("user can see their current team", () => {
```

Comment:
The way this test is set up is artificial and weird, but it's the only way I found to keep the rendered state in sync with the values passed to the `useQuery` mock.

You can see the way it was failing in this commit (rendered state seemed to actually be in sync with the `useQuery` mock in the *previous* `it` block): https://github.com/aiven/klaw/pull/791/commits/3312942d4f076e7f864c02bfbe31d939648a5cd1